artemis-chart/README.md

```diff
@@ -0,0 +1 @@
+This chart was created by Kompose
```

Comment:
Maybe here some instructions on how to set this up? N steps needed to set everything up for helm (and artemis over helm).